docs/tutorials/exploring_bdk_rn.md

```diff
@@ -0,0 +1,655 @@
+description: "A guide to using bdk-rn for building bitcoin apps"
+authors:
+    - Bitcoin Zavior
+date: "2022-08-05"
+tags: ["bdk-rn", "bdk", "tutorial", "guide", "wallet"]
+---
+
+## Introduction
+
+The **BitcoinDevkit**'s **React Native** library, bdk-rn enables building bitcoin applications for Android and iOS mobile platforms. Using bdk-rn does not require knowledge of the underlying bitcoin or BDK API, and using bdk-rn is similar to using any other RN module. 
+
+In this tutorial, we will explore bdk-rn usage and the API it provides. This guide will walk through the development process and code for making a bitcoin application. The bitcoin application we create will be a non-custodial HD Wallet. The application will have the functionality to create a new wallet or restore from a known mnemonic seed phrase. The application will also be able to interact with the bitcoin network to sync UTXOs from new blocks and broadcast transactions.
+
+The tutorial will focus on bitcoin and bdk-rn concepts and API. So it will gloss over react native related aspects. The code for this tutorial is available on GitHub at https://github.com/LtbLightning/AwesomeBitcoinApp
+
+<img src="./exploring_bdk_rn/bdk_rn_complete_app.png" alt="BDK RN Quick Start" style="display: block; margin: 0 auto; zoom: 50%;" />
+
+### Prerequisites
+
+In order to use bdk-rn in a React Native App, a React Native development environment is required. Please refer to resources out there on the internet if you need to set this up, here is one of many good resources to guide you https://reactnative.dev/docs/environment-setup
+
+### Bitcoin Basics
+
+The bitcoin concepts used in this blog post are detailed and explained very well in external bitcoin resources. Here are some links for reference:
+
+Mastering Bitcoin(HD Wallet chapter): https://github.com/bitcoinbook/bitcoinbook/blob/develop/ch04.asciidoc Bitcoin Output Descriptors from GitHub: https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md
+
+Now let's jump into Bitcoin Dev Kit
+
+## Bitcoin Dev Kit and bdk-rn
+
+`bdk-rn` is a React Native library of Bitcoin Dev Kit(BDK) for building React Native Apps.
+It encapsulates all of the low-level APIs and methods for BDK and exposes them in a react native context. To use BDK in React Native(RN) apps only the `bdk-rn` module is required. `Bdk-rn` can be used like any other react native library and is available in public package managers.
+
+## Getting Started
+
+Although we won't delve deep into RN we will focus more on bitcoin and bdk-rn, however, some rudimentary RN setup is required, especially a basic RN app to add our code.
+
+ start by creating a new RN project.
+
+`npx react-native init AwesomeBitcoinApp`
+
+If this fails end is an error on an M1 Mac please use
+ `arch -x86_64 pod install --repo-update`
+
+Once done lets `cd` into the new project and run the basic RN app thats created
+
+```shell
+cd AwesomeBitcoinApp
+yarn ios
```

Comment:
Sometimes you need to cd into the iOS folder and do a pod install.

Reply:
When you first create the App `pod install` is not required.
After adding `bdk-rn `or other modules a pod install is required, which is mentioned in the following paragraph.